.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,15 +5,15 @@
 
 ## Does this PR introduce a breaking change?
 
-read [What is a breaking change ?](https://mozaic.adeo.cloud/Contributing/Prerequisite/GitConventions/#breaking-changes-)
+<!-- Read [what is a breaking change?](https://mozaic.adeo.cloud/Contributing/Developers/GitConventions/#breaking-changes-) -->
```

Comment:
This text should not be commented on.
Can you remove the `<!-- -->` please?

Reply:
@tiloyi what about having it directly into the title?

Reply:
Indeed, it seems good to me to do that :)